site/pages/join-us/index.js

```diff
@@ -103,46 +80,84 @@ const apply = {
 const vacancies = {
   type: 'Title2',
   props: {
-    children: 'Current Vacancies',
+    children: <span className={styles.vacancyHeader}>Current Vacancies</span>,
   },
 };
 
-export default function JoinUs({ jobs }) {
-  return (
-    <div className={styles.background}>
+class HeaderContainer extends React.Component {
+  static showPicture() {
+    const MEDIUM_BREAKPOINT = '(min-width: 690px)';
+    return !window.matchMedia(MEDIUM_BREAKPOINT).matches;
+  }
+
+  state = {
+    show: HeaderContainer.showPicture(),
+  };
+
+  componentDidMount() {
+    window.addEventListener('resize', this.onResize);
+  }
```

Comment:
When the component unmounts, we need to remove the listener - or else when the component unmounts, the listener will be kept in memory.
```
componentWillUnmount: function() {
        window.removeEventListener("resize", this.onResize);
    }
});
```

Reply:
done, thanks